README.rst

```diff
@@ -250,6 +250,16 @@ Weekly Developer meeting
 * Weekly, Wednesday, 8:00 am PT, 11:00 am ET, 5:00 pm CEST
 * `Join zoom <https://zoom.us/j/596609673>`_
 
+Governance
+----------
+The Cilium project is governed by a group of `maintainers and commiters <https://github.com/cilium/cilium/blob/master/MAINTAINERS.md.>`_.
```

Comment:
Broken link because of the trailing period.

```suggestion
The Cilium project is governed by a group of `Maintainers and Committers <https://raw.githubusercontent.com/cilium/cilium/master/MAINTAINERS.md>`__.
```
I'm suggesting the raw file because this is what is linked from the Governance page, but if you prefer to keep the GitHub-rendered Markdown that's fine as well.